README.md

```diff
@@ -764,6 +764,50 @@ For more detailed instructions on using the Toolbox Core SDK, see the
 </blockquote>
 </details>
 
+### Using Toolbox with Gemini CLI Extensisions
+
+[Gemini CLI extensions][gemini-cli-extensions] provide tools to interact
+directly with your data sources from command line. Below is a list of Gemini CLI
+extensions that are built on top of **Toolbox**. They allow you to interact with
+your data sources through pre-defined or custom tools with natural language.
+Click into the link to see detailed instructions on their usage.
+
+To use **custom** tools with Gemini CLI:
+
+- [MCP Toolbox](https://github.com/gemini-cli-extensions/mcp-toolbox)
+
+To use **pre-defined** tools with Gemini CLI:
```

Comment:
Can this be "prebuilt configurations"? Let's also link to the stuff for prebuilt configs